static/window/styles/core.css

```diff
@@ -52,7 +52,9 @@ body {
   width: 1rem;
   height: 1rem;
 }
-
+.icon--magnifying-glass {
```

Comment:
This applies to all magnifying icons and not just the one in the search bar.

Use something like ``.header__search__icon`` or ``.header__search__icon.icon--magnifying-glass`` instead (and remember to move it)